src/index.html

```diff
@@ -11,6 +11,19 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="https://www.google.com.ua/?hl=ru" method="get"
+    target="_blank" data-qa="big"
+    class="big-form">
+      <input type="text" placeholder="Try &quot;Los Angeles&quot;"
+      class="big-search-input" data-qa="keypress">
+    </form>
```

Comment:
```suggestion
    <form
      action="https://www.google.com.ua/?hl=ru" method="get"
      target="_blank"
      data-qa="big"
      class="big-form"
    >
      <input
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        class="big-search-input"
        data-qa="keypress"
      >
    </form>
```
same for second form

Reply:
done